pkg/virt-controller/services/net_binding.go

```diff
@@ -51,3 +53,24 @@ func NetBindingPluginSidecarList(vmi *v1.VirtualMachineInstance, config *v1.Kube
 	}
 	return pluginSidecars, nil
 }
+
+const SlirpNetworkBindingPluginName = "slirp"
+
+func ReadNetBindingPluginConfiguration(kvConfig *v1.KubeVirtConfiguration, pluginName string) (v1.InterfaceBindingPlugin, error) {
+	if kvConfig != nil && kvConfig.NetworkConfiguration != nil && kvConfig.NetworkConfiguration.Binding != nil {
+		if plugin, exist := kvConfig.NetworkConfiguration.Binding[pluginName]; exist {
+			return plugin, nil
+		}
+	}
+
+	// TODO: in case no Slirp network binding plugin is registered (i.e.: spesifed in in Kubevirt config) use the
+	// following default image to prevent newly created Slirp VMs from handing, and reduce friction for users who didnt
+	// registered and image yet. This workaround should be removed by the next Kubevirt release v1.2.0.
+	const defaultImage = "quay.io/kubevirt/network-slirp-binding:20230830_638c60fc8"
+	if pluginName == SlirpNetworkBindingPluginName {
+		log.Log.Warningf("no Slirp network binding plugin image is set in Kubevirt config, using %q sidecar image for Slirp network binding configuration", defaultImage)
+		return v1.InterfaceBindingPlugin{SidecarImage: defaultImage}, nil
+	}
+
+	return v1.InterfaceBindingPlugin{}, fmt.Errorf("couldn't find %s network binding plugin configuration, make sure its specified in Kubevirt config", pluginName)
```

Comment:
I do not see the usefulness to feedback that the lookup did not find anything using an error.
It can just return a `nil` like other lookups and the error can be composed at the caller.